liz-areizaga/summarizer
# Backend setup

## To test:

* resolve #9 
* resolve #8 
* `cd frontend`
* `npm i`
* `npm start`
* Expect browser to open, and return summary in console log after pressing submit
